app/src/main/java/ca/chronofit/chrono/main/circuit/CircuitTimerActivity.kt

```diff
@@ -356,4 +361,35 @@ class CircuitTimerActivity : AppCompatActivity() {
             0
         }
     }
+
+    private fun createNotification(time: Float) {
+        val builder =
+            NotificationCompat.Builder(this, getString(R.string.timer_notification_channel_id))
+                .setSmallIcon(R.drawable.ic_notification_logo)
+                .setContentTitle("Notification test")
```

Comment:
Is this needed right now?

Reply:
Commented out the part that issues the notification.